### ISSUE TITLE
Unable to use module with debian/jessie and 1labs repo

### ISSUE THREAD

**Issue by sts** [djjudas21/puppet-freeradius#77]
Due to the validate_re failing the module, on Debian/Jessie I'm unable to bootstrap a working Freeradius from 1labs repo. Can this be replaced with a warning instead? 

```
 apt::source { '1labs':
    comment  => "1Labs Freeradius ${release} repository",
    location => 'http://pkg.1labs.ch/debian',
    release  => "${release}-1labs",
    repos    => 'fr30-release',
    key      => {
      id     => '13C0A89B5C671AAB72DC07B47DADAF130A2FCD77',
    },
  } ->

  class { '::freeradius':
    preserve_mods => false,
    utils_support => true,
  }
```

**Comment by djjudas21** [comment]
What string are you trying to match? I don't have a Debian system to test on.

**Comment by sts** [comment]
The issue is this line:

https://github.com/djjudas21/puppet-freeradius/blob/1ada5b7bbc0e9b0f6f9c06dba1e51fbb355c0fb0/manifests/init.pp#L22

Which could be replaced by something like:

```puppet
if $freeradius::fr_version =~ /^3/ {
  notify { "This module is only compatible with FreeRADIUS 3.": } 
}
```

**Comment by djjudas21** [comment]
Fixed in e6cefed49097c01ddaeb3db65d19f776b3817897
